### ISSUE TITLE
Add filename to log information

### ISSUE THREAD

**Issue by razbakov** [firegento/firegento-logger#77]
Log function `Mage::log($message, $level = null, $file = '', $forceLog = false)`
has 3d parameter `$file`.

We should add `$file` to log as well.

This will help to group logs.

**Comment by razbakov** [comment]
@daim2k5 do you know how to do it?

**Comment by datenbrille** [comment]
The parameter will tell the log system in which file to log. But with some logger it make no sense, because there are no files. What is the intention to have the file name in the message?

**Comment by razbakov** [comment]
Different modules write debug information in specific files:

```
Mage::log('debug info', 7, 'payment_debug.log');
Mage::log('my custom debug message', 7, 'check_it_later.log');
```

And if you use Logstash then log entry is just json object of different values:
https://github.com/firegento/firegento-logger/blob/develop/src/app/code/community/FireGento/Logger/Model/Logstash.php#L90

I already added some useful info in #76 but my team also want to have log file name here.
Any idea how to do it?

**Comment by erikmulder** [comment]
Add filename of logfile to json data: 
The filename is passed as an argument to the writer classes. In for example the class 
FireGento_Logger_Model_Logstash you can set the file name in the constructor and use it 
when you're building the json string in the method buildJSONMessage.

**Comment by colinmollenhour** [comment]
Looks like this was addressed by #78
